### ISSUE TITLE
Blank Edit / New Post or Page Screen after Update to Wordpress 6.2

### ISSUE THREAD

**Issue by mlabmedia** [htmlburger/carbon-fields#1158]
## Version

  - Carbon Fields:3.5.2
  - WordPress: 6.2
  - PHP: 8.0
  
  After updating to Wordpress 6.2 I am no longer able to access the edit screen or create new page / posts. The issue seems to be something with metaboxes.min.js . I have posted a screen shot below of the error I am getting in console.

![carbonfields](https://user-images.githubusercontent.com/17283389/229002593-ad23173b-fdb8-4cd0-8626-5f99e403fbf3.png)

I have attempted deleting these two files stated in the error above to just test if it would allow me to access the edit screen, and it did work but is not a solution unfortunately.

Downgarding to Wordpress 6.1 does work fine and I have no issues.
```

## Steps to Reproduce the Problem

  1. Update to Wordpress 6.2

**Comment by ste6412** [comment]
I downgraded to carbon fields v3.4 and it fixed the blank screen issue for me

**Comment by vncnt-mb** [comment]
Hello,
Got this error today and I found a "workaround".

Since i remove all containers contains
` ->where( 'post_template', '=', 'template-[...].php' )`

editor comes back.

I don't know if this is the real cause but in any case, it works

if it can help you or the devs :)

**Comment by sergromanenko1** [comment]
Hello, I have the same problem, after updating to Wordpress 6.2.
The broblem is with the Block Editor Gutenberg, but not with the Classic Editor.
If you don't depend on the Block Editor Gutenberg, you can install the Advanced Editor Tools plugin and activate option https://disk.yandex.ru/i/7GQ7duP4VTdNfQ

**Comment by HeroBanana** [comment]
The main issue is React 18. The elements no longer support directly accessing root through _reactRootContainer. Working on a root registry. PR coming.

**Comment by HTMLBurger-NG** [comment]
Hi @mlabmedia,

Thank you for your feedback.
We've recently pushed a new release v3.6.0 where we've fixed the issues with WordPress 6.2.

**Comment by plapointe** [comment]
Any updates on this issue?
